proto/catalog.proto

```diff
@@ -58,6 +58,16 @@ message Table {
   bool appendonly = 14;
 }
 
+message InternalStateTable {
+  uint32 id = 1;
+  uint32 schema_id = 2;
+  uint32 database_id = 3;
```

Comment:
`schema_id` and `database_id` are not necessary. And I think we may still need `associated_table_id` field, which is necessary for permission check when query on internal state.